--- origsrc/atk-1.1.4/ext/atk/rbatk.h	2012-08-22 19:23:01.017299600 -0500
+++ src/atk-1.1.4/ext/atk/rbatk.h	2012-08-22 19:24:39.898955300 -0500
@@ -28,7 +28,7 @@
 #include "rbatkversion.h"
 #include "rbatkconversions.h"
 
-#if defined(G_PLATFORM_WIN32) && !defined(RUBY_ATK_STATIC_COMPILATION)
+#if defined(G_OS_WIN32) && !defined(RUBY_ATK_STATIC_COMPILATION)
 #  ifdef RUBY_ATK_COMPILATION
 #    define RUBY_ATK_VAR __declspec(dllexport)
 #  else
